srcpkgs/disklow/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'disklow'
+pkgname=disklow
+version=1.2
+revision=1
+depends="perl perl-Filesys-Df perl-Config-General perl-Mail-Sendmail perl-Net-SMTP-SSL perl-Authen-SASL"
```

Comment:
I think you don't need to depend on `perl` directly once you depend on all those.

Reply:
Removed "perl"